### ISSUE TITLE
please update brillantLabs discord invite link

### ISSUE THREAD

**Issue by parthiv11** [brilliantlabsAR/docs#66]
old link is expired please update it @siliconwitch

**Comment by siliconwitch** [comment]
Link works fine for me. Do you mean one on the docs home page?

<img width="1095" alt="Screenshot 2024-09-04 at 10 31 15" src="https://github.com/user-attachments/assets/cb41423a-6a37-4d8c-8948-cd0023abbf52">

**Comment by parthiv11** [comment]
![Screenshot_20240904_140509](https://github.com/user-attachments/assets/eb4135bb-4fb7-45f0-9622-79fc65221eeb)

**Comment by parthiv11** [comment]
Now working

**Comment by siliconwitch** [comment]
okay now?

**Comment by parthiv11** [comment]
No

**Comment by parthiv11** [comment]
On website, GitHub page and documentation all discord links shows same error

**Comment by siliconwitch** [comment]
What's your Discord username?

**Comment by parthiv11** [comment]
parthiv1119

**Comment by siliconwitch** [comment]
Does this link work for you? https://discord.gg/5jfvRe7v

**Comment by parthiv11** [comment]
No

**Comment by siliconwitch** [comment]
Then I don't know. It's a fresh link. All the links on the website and docs are working for everyone else. Maybe something with your discord account or connection

**Comment by siliconwitch** [comment]
I asked several people to test and it seems okay

**Comment by siliconwitch** [comment]
Try 3G/4G instead of your wifi

**Comment by parthiv11** [comment]
Yes you are right, it works for others.

But not for me

**Comment by parthiv11** [comment]
If you can see that I am blocked

**Comment by siliconwitch** [comment]
Yeah, I checked. You're not on the ban list so that shouldn't be the issue

**Comment by parthiv11** [comment]
tried everything to join from my account, but not working

thank you @siliconwitch 

i will try to solve and join as early as possible.

currently closing this issue as this is not related docs problem

**Comment by parthiv11** [comment]
@siliconwitch I know you have checked block list already.

I asked discord support team and I request you to please checkout as per below image shows 
![Screenshot_2024_0909_214740](https://github.com/user-attachments/assets/4bb6ef18-dafa-4de5-949f-ea8d186abe8d)

**Comment by parthiv11** [comment]
@siliconwitch have you checked

**Comment by siliconwitch** [comment]
You're not on the ban list. Did you have any previous accounts? Maybe try a different discord account

**Comment by parthiv11** [comment]
I had a account which got a hacked, then I deleted that account and reopened with same username

**Comment by parthiv11** [comment]
![discord](https://i.ibb.co/NZyPtcq/Screenshot-20240913-193928.jpg)
